### ISSUE TITLE
question about u-net

### ISSUE THREAD

**Issue by gztleo** [zhixuhao/unet#30]
Dear， I think your architecture of u-net is not correct for the beginning layers. I use it to train the same dataset, but the loss doesn't converge.Could you upload your trained model?Thank you!

**Comment by gabriel1984** [comment]
for keras has updated to 2.1.5 now
